modules/openshift-cluster-maximums-major-releases.adoc

```diff
@@ -8,6 +8,11 @@
 Tested Cloud Platforms for {product-title} 3.x: {rh-openstack-first}, Amazon Web Services and Microsoft Azure.
 Tested Cloud Platforms for {product-title} 4.x: Amazon Web Services, Microsoft Azure and Google Cloud Platform.
 
+[NOTE]
+====
+Some limits were tested independently, therefore a comparison between rows should not be made.
```

Comment:
Consider active - "therefore do not compare between rows" or "therefore do not make a comparison between rows"